plumgrid_playbooks/plumgrid_preinstall_check.yml

```diff
@@ -24,12 +24,14 @@
       stat:
         path: "{{ pg_license_path }}"
       register: lic_present
-      failed_when: install_license == True and (lic_present.stat.exists == False or lic_present.stat.size == 0)
```

Comment:
no point to use `==` to check it against `True` will evaluate if it's value is `True`